tests/tests_zone.yml

```diff
@@ -41,6 +41,10 @@
                 forward_port: [ '447/tcp;;1.2.3.4',
                                 '448/tcp;;1.2.3.5' ],
                 state: 'enabled' }
+            - { zone: 'customzone',
```

Comment:
unless the value is a single line for compactness, it is better to use the standard yaml layout for a hash value instead of the json style - so
```
  - zone: customzone
    masquerade: yes
    permanent: yes
    state: enabled
```
I see that the other values in tests_zone use `{` and `}` but they shouldn't 
NOTE that in some cases using the json style is a good thing e.g. for the array values like
```
                service: [ 'tftp', 'ftp' ],
                port: [ '443/tcp', '443/udp' ],
```
instead of the yaml style
```
                service:
                  - tftp
                  - ftp
                port:
                 - '443/tcp'
                 - '443/udp'
 ```
because the json style makes the code more readable

also note that there is a big difference between `param: yes` and `param: 'yes'` - the former is a boolean value, and the latter is a string value - they are not the same value.  Since parameters like `masquerade` and `permanent` are boolean valued, you must use `yes` and not `'yes'`